doc/Security.xml

```diff
@@ -4,19 +4,19 @@
   <info>
     <title>Advanced Security Service Specification</title>
     <titleabbrev>Security Configuration</titleabbrev>
-    <releaseinfo>22.06</releaseinfo>
+    <releaseinfo>23.06</releaseinfo>
     <author>
       <orgname>ONVIF™</orgname>
       <uri>www.onvif.org</uri>
     </author>
-    <pubdate>June 2022</pubdate>
+    <pubdate>June 2023</pubdate>
     <mediaobject>
       <imageobject>
         <imagedata fileref="media/logo.png" contentwidth="60mm"/>
       </imageobject>
     </mediaobject>
     <copyright>
-      <year>2008-2022</year>
+      <year>2008-2023</year>
```

Comment:
2008?

Change 2008-2023 to 2023?

Reply:
is the security spec really copyright 2023? Should we generally update the copyright year to latest change year or should we keep the historical way?